checks/raw/security_policy.go

```diff
@@ -61,7 +61,13 @@ func SecurityPolicy(c *checker.CheckRequest) (checker.SecurityPolicyData, error)
 		return checker.SecurityPolicyData{}, err
 	}
 
-	// Return raw results.
+	// Modify the path to include org name and return raw results.
+	for i := range files {
+		// Add the client URI before the path
+		files[i].Path = dotGitHubClient.URI() + "/" + files[i].Path
```

Comment:
Use https://pkg.go.dev/path#Join to join paths.

Reply:
Modified, thanks